models/safe/base/safe_base_singletons.sql

```diff
@@ -0,0 +1,16 @@
+{{
+    config(
+        materialized='table',
+        tags = ['dunesql'],
+        alias= alias('singletons'),
+        post_hook='{{ expose_spells(\'["base"]\',
+                                    "project",
+                                    "safe",
+                                    \'["danielpartida"]\') }}'
+    )
+}}
+
+
+-- Fetch all known singleton/mastercopy addresses used via factories.
+select distinct singleton as address
+from {{ source('gnosis_safe_base', 'GnosisSafeProxyFactoryv_1_3_0_evt_ProxyCreation') }}
```

Comment:
The source table name is tricky here, there is a letter `v` after `Factory`:
https://dune.com/queries/2965816?category=decoded_project&blockchain=base&namespace=gnosis_safe&contract=GnosisSafeProxyFactoryv_1_3_0&blockchains=base

Reply:
likely missed `_` between the `Factory` and `_1_3_0` in the decoding process

if ever fixed there, we'll fix here, but this works for now